pkg/apiserver/api/v1/offering.proto

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+
+import "types.proto";
+
+message Offering {
+  string name = 1;
+  string tenant = 2;
+  OfferingMetadata metadata = 3;
+  ObjectReference provider = 4;
```

Comment:
This doesn't feel right; why cannot this be a string? It makes easier API usage, more straightforward. Same as with tenant.

Reply:
I just want to keep consistent with our catalogv1alpha1.Offering definition